docker/jenkins/javascript.sh

```diff
@@ -40,13 +40,13 @@ function run_tests {
     docker ps -a | grep $COMPOSE_PROJECT_NAME | awk '{print $0}'
 
     docker-compose -f $COMPOSE_FILE_LOC -p $COMPOSE_PROJECT_NAME run --name ${TEST_CONTAINER_REF}_test \
-                frontend_tester npm run test:ci
+                frontend_tester npm run test:ci || true
```

Comment:
Should we add comments explaining why the `|| true` are needed? It isn't immediately obvious.